project/Dependencies.scala

```diff
@@ -2,116 +2,189 @@ import sbt._
-    "com.typesafe.akka"      %% "akka-testkit"      % akkaVersion     % Test exclude ("com.typesafe", "config"),
-    "com.typesafe.akka"      %% "akka-http-testkit" % akkaHttpVersion % Test exclude ("com.typesafe", "config"),
-    "org.scalatest"          %% "scalatest"         % "3.0.5"         % Test,
-    "com.softwaremill.diffx" %% "diffx-scalatest"   % "0.3.3"         % Test,
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
-  )
+  private val scalaTestCompile = Seq("org.scalatest" %% "scalatest" % "3.0.5") // Dedicated for common-testkit
+  private val scalaTest = scalaTestCompile.map(_     % Test)
+  private val scalaTestJson = Seq("com.stephenn"     %% "scalatest-json-jsonassert" % "0.0.3" % Test)
 
-  val conseilLorreInclude = Seq(
-    "org.scalamock" %% "scalamock" % "4.1.0" % Test,
-    "org.scalatest" %% "scalatest" % "3.0.5" % Test,
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
+  private val scalaMock = Seq("org.scalamock" %% "scalamock" % "4.1.0" % Test)
+
+  private val postgresTestContainerCompile = Seq("org.testcontainers"    % "postgresql" % "1.12.3")
+  private val postgresTestContainer = postgresTestContainerCompile.map(_ % Test)
+
+  private val diffX = Seq("com.softwaremill.diffx" %% "diffx-scalatest" % "0.3.3" % Test)
+
+  private val apacheCommonsText = Seq("org.apache.commons" % "commons-text" % "1.7")
+
+  private val radixTree = Seq("com.rklaehn" %% "radixtree" % "0.5.1")
+
+  private val jna = Seq(
+    "com.muquit.libsodiumjna" % "libsodium-jna" % "1.0.4" exclude ("org.slf4j", "slf4j-log4j12") exclude ("org.slf4j", "slf4j-api"),
+    "net.java.dev.jna"        % "jna"           % "5.5.0" //see https://github.com/muquit/libsodium-jna/#update-your-projects-pomxml
   )
 
-  val conseilSmokeTestsInclude = Seq(
-    "com.typesafe"  % "config"               % "1.3.3",
-    "org.typelevel" %% "cats-core"           % catsVersion,
-    "org.http4s"    %% "http4s-blaze-client" % http4sVersion,
-    "org.http4s"    %% "http4s-dsl"          % http4sVersion,
-    "org.http4s"    %% "http4s-circe"        % http4sVersion,
-    "io.circe"      %% "circe-core"          % circeVersion,
-    "io.circe"      %% "circe-parser"        % circeVersion
+  private val chimney = Seq("io.scalaland" %% "chimney" % "0.3.1")
+
+  private val bitcoin = Seq("fr.acinq" %% "bitcoin-lib" % "0.9.18-SNAPSHOT")
+
+  private val scorex = Seq(
+    "org.scorexfoundation" %% "scrypto"     % "2.1.7",
+    "org.scorexfoundation" %% "scorex-util" % "0.1.6"
   )
 
+  //List of dependencies that are not really used (?) - to be removed
+  ////      "com.chuusai"             %% "shapeless"   % "2.3.3",
```

Comment:
I'm not totally sure if it's not at least marginally (like 1 line of code) used anywhere.

The point here is that shapeless is used by many of our dependencies anyway, and I prefer to have a direct specific dependency if we use it, rather than by transitivity, if you get my meaning

Reply:
I checked if packages from this class are imported anywhere and I believe they are not, thus I decided to comment them out and eventually remove.